docs/rst/manual/getting_started/configuration/poly_new/guide.rst

```diff
@@ -0,0 +1,291 @@
+Creating a config
+=================
+Ert runs a set of simulations with prior set of parameters sampled from distributions. The results of the simulations are then given to an optimization algorithm which uses a set of observations to tune the parameters in order to make a next run of the simulations produce results that fit better with the set of observations.
+
+To understand the basics of how to configure and use ERT, we will walk through how to set up a simple project that uses a simple polynomial evaluation as the forward model.
+
+Minimal config
+--------------
+In order to test that our setup is working, we will create the minimal configuration we need in order to lauch the Graphical user interface and run an empty experiment that does nothing. First create a folder with the name "poly_example". 
+
+Create a config
+***************
+Then create a file "poly.ert" inside the folder with the following content:
+
+    .. literalinclude:: minimal/poly.ert
+
+* :ref:`JOBNAME <jobname>` Specifies the name for each simulation. The %d will be replaced with the simulation number, so that the jobs will be called poly_0, poly_1, etc.
+* :ref:`NUM_REALIZATIONS <num_realizations>` We have to specify how many simulation we want. The word realization relates to the mathematical concept of an actual observation from a distribution.
```

Comment:
Missing a `s` in simulations. Would reword `The word realization relates to the mathematical concept of an actual observation from a distribution.`,  as a realization is one member of an ensamble, so realizations sum up to ensamble size.

Reply:
Removed the sentence and added some more info in the first paragraph